java.lang.NoClassDefFoundError: com/fasterxml/jackson/databind/ObjectMapper

For the Form containing external geojson instance, I am getting following error:

```
Exception in thread "main" java.lang.NoClassDefFoundError: com/fasterxml/jackson/databind/ObjectMapper
        at org.javarosa.core.model.instance.geojson.GeoJsonExternalInstance.parse(GeoJsonExternalInstance.java:35)
        at org.javarosa.xform.parse.ExternalInstanceParser.parse(ExternalInstanceParser.java:40)
        at org.javarosa.xform.util.XFormUtils.getExternalInstance(XFormUtils.java:183)
        at org.javarosa.core.model.instance.ExternalDataInstance.parseExternalInstance(ExternalDataInstance.java:81)
        at org.javarosa.core.model.instance.ExternalDataInstance.build(ExternalDataInstance.java:65)
        at org.javarosa.xform.parse.XFormParser.parseDoc(XFormParser.java:579)
        at org.javarosa.xform.parse.XFormParser.parse(XFormParser.java:417)
        at org.javarosa.xform.parse.XFormParser.parse(XFormParser.java:385)
        at org.javarosa.xform.util.XFormUtils.getFormFromInputStream(XFormUtils.java:111)
        at org.javarosa.xform.util.XFormUtils.getFormFromInputStream(XFormUtils.java:92)
        at org.opendatakit.validate.FormValidator.validate(FormValidator.java:462)
        at org.opendatakit.validate.FormValidator.validate(FormValidator.java:420)
        at org.opendatakit.validate.FormValidator.validate(FormValidator.java:392)
        at org.opendatakit.validate.FormValidator.validateAndExitWithErrorCode(FormValidator.java:349)
        at org.opendatakit.validate.FormValidator.main(FormValidator.java:141)
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.databind.ObjectMapper
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
        ... 15 more
```

#### Software versions 
Validate v1.19.1
Java v22.0.2 2024-07-16
operating system: MacOS 14.5 (23F79)

Issue is reproducible in pyxform-http and xlsform-online as well.

#### Problem description

Application crashes when Form contains external geoJSON

#### Steps to reproduce the problem

`java -jar ODK-Validate-v1.19.1.jar ~/Downloads/all-widgets.xml `

#### Expected behavior

Forms with external getJSON should be gracefully handled.

#### Other information 

Example Form:
```
<?xml version="1.0"?>
<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml"
  xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema"
  xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms"
  xmlns:odk="http://www.opendatakit.org/xforms">
  <h:head>
    <h:title>Simple</h:title>
    <model odk:xforms-version="1.0.0">
      <instance>
        <data id="Simple">
          <map_widget />
        </data>
      </instance>
      <instance id="cities" src="jr://file/cities.geojson" />
      <bind nodeset="/data/map_widget" type="string" />
    </model>
  </h:head>
  <h:body>
    <select1 appearance="map" ref="/data/map_widget">
      <label>Map widget</label>
      <itemset nodeset="instance('cities')/root/item">
        <value ref="id" />
        <label ref="title" />
      </itemset>
    </select1>
  </h:body>
</h:html>
```

--- a/build.gradle
+++ b/build.gradle
@@ -28,9 +28,7 @@ sourceCompatibility = '1.8'
 
 dependencies {
     compile group: 'net.sf.kxml', name: 'kxml2', version: '2.3.0'
-    compile (group: 'org.getodk', name: 'javarosa', version: '4.4.1') {
-        exclude group: "com.fasterxml.jackson.core" // Validate doesn't validate attached geoJSON files
-    }
+    compile (group: 'org.getodk', name: 'javarosa', version: '4.4.1')
     compile group: 'org.slf4j', name: 'slf4j-nop', version: '1.7.25'
     testCompile 'junit:junit:4.13.2'
     testCompile 'org.hamcrest:hamcrest-all:1.3'

